src/lib/selection/media/html-selector.js

```diff
@@ -211,6 +228,9 @@ export default class HTMLSelector extends MediaSelector {
     const selection = HTMLSelector.getSelection(this.target)
 
     let anchor = selection.anchorNode // A node where is a beginning of a selection
+    if (!anchor) {
+      return undefined
```

Comment:
I think we can just `return`, it'll be the same

Reply:
Fixed